src/scripts/templates/messages.py

```diff
@@ -0,0 +1,4 @@
+from colors import _bcolors
+
+GENERATING = "%sGenerating tests%s" % (_bcolors.okblue, _bcolors.endc)
+DONE = "%sDone%s" %(_bcolors.okblue, _bcolors.endc)
```

Comment:
Put the whole contents of this file into the `utils.py` like in #2 .

Reply:
Let's move this into a messages class in util.py